### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by codeaye**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,15 @@
 {
   "name": "mathjax-angular-project",
   "version": "1.1.0",
+  "description": "A quick way to implement Mathjax v3 into your Angular project!",
```
Comment (sajivkumar) [inline package.json:4]: update in both readme,nd also

Comment (codeaye) [reply package.json:4]: ? what should i do in readme

Comment (sajivkumar) [reply package.json:4]: so no update required in readme.md ?

Comment (sajivkumar) [reply package.json:4]: hopefully all cahnges that you did in this package.json would be done in package.json inside mathjax-lib folder also

Comment (codeaye) [reply package.json:4]: Ok check again

**Pull request merged**